src/views/DocumentationTopic.vue

```diff
@@ -9,74 +9,88 @@
 -->
 
 <template>
-  <div :class="{ 'modal-open': quickNavigationStore.state.showQuickNavigation }">
-    <div v-show="quickNavigationStore.state.showQuickNavigation">
-      <QuickNavigationModal />
-    </div>
-    <CodeTheme class="doc-topic-view">
-      <template v-if="topicData">
-        <component
-          :is="enableNavigator ? 'AdjustableSidebarWidth' : 'StaticContentWidth'"
-          v-bind="sidebarProps"
-          v-on="sidebarListeners"
+  <CodeTheme class="doc-topic-view">
+    <template v-if="topicData">
+      <component
+        :is="enableNavigator ? 'AdjustableSidebarWidth' : 'StaticContentWidth'"
+        v-bind="sidebarProps"
+        v-on="sidebarListeners"
+      >
+        <PortalTarget name="modal-destination" multiple />
+        <template #aside="{ scrollLockID, breakpoint }">
+          <NavigatorDataProvider
+            :interface-language="topicProps.interfaceLanguage"
+            :technology="technology"
+            :api-changes-version="store.state.selectedAPIChangesVersion"
+            ref="NavigatorDataProvider"
+          >
+            <template #default="slotProps">
+              <div class="doc-topic-aside">
+                <QuickNavigationModal
+                  :children="slotProps.flatChildren"
+                  :showQuickNavigationModal="showQuickNavigationModal"
+                  @update:showQuickNavigationModal="(show) => showQuickNavigationModal = show"
```

Comment:
Same in here, these 2 lines can be replaced by just:
```suggestion
                  :showQuickNavigationModal.sync="showQuickNavigationModal"
```

Reply:
Resolved [0b98f4c](https://github.com/apple/swift-docc-render/pull/396/commits/0b98f4c4a9201ff5868f778dc9a0770905b1eba2)